File: myonlinefm/src/main/java/com/sec/myonlinefm/onlineinfolistener/ObserverListener.java
package com.sec.myonlinefm.onlineinfolistener;

import com.sec.myonlinefm.data.Station;
import com.sec.myonlinefm.data.StationProgram;

import java.util.List;
import java.util.Map;

/**
 * Created by SRC-TJ-MM-BinYang on 2018/1/31.
 */

public interface ObserverListener {
    void observerLiveRadioLocalUpData(List<Station> mStations, Map<Integer, List<StationProgram>> map);
    void observerLiveRadioCenterUpData(List<Station> mStations, Map<Integer, List<StationProgram>> map);
    void observerUpDataDifferentInfo(List<Station> mStations, Map<Integer, List<StationProgram>> map);
}

File: myonlinefm/src/main/java/com/sec/myonlinefm/onlineinfolistener/ObserverListener.kt
package com.sec.myonlinefm.onlineinfolistener

import com.sec.myonlinefm.data.StationProgram
import com.sec.myonlinefm.data.Station


/**
 * Created by SRC-TJ-MM-BinYang on 2018/1/31.
 */
interface ObserverListener {
    open fun observerLiveRadioLocalUpData(mStations: MutableList<Station?>?, map: MutableMap<Int?, MutableList<StationProgram?>?>?)
    open fun observerLiveRadioCenterUpData(mStations: MutableList<Station?>?, map: MutableMap<Int?, MutableList<StationProgram?>?>?)
    open fun observerUpDataDifferentInfo(mStations: MutableList<Station?>?, map: MutableMap<Int?, MutableList<StationProgram?>?>?)
}